### PR TITLE
dbsp: Add README for crates.io.

### PR DESCRIPTION
**Pull request by blp**

### CODE REVIEW

File: crates/dbsp/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# DBSP
+
+DBSP is a computational engine for continuous analysis of changing
```
Comment (ryzhyk) [inline crates/dbsp/README.md:3]:
```suggestion
Database Stream Processor (DBSP) is a computational engine for continuous analysis of changing
```

File: crates/dbsp/README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+# Resources
+
+## Learning Materials
+
+- The [tutorial] walks through a series of simple examples.
+
+- The [`circuit` module documentation] provides a catalog of
+  computational elements.
+
+- More sophisticated uses can be found as [examples] and [benchmarks]
```
Comment (ryzhyk) [inline crates/dbsp/README.md:23]: We probably also want a link to the paper.

**Review by ryzhyk — APPROVED** [review]

File: crates/dbsp/README.md (hunk 1 above)
Comment (blp) [reply crates/dbsp/README.md]: Done.

File: crates/dbsp/README.md (hunk 2 above)
Comment (blp) [reply crates/dbsp/README.md:23]: Added.

**Pull request merged**